shell/pages/c/_cluster/apps/charts/chart.vue

```diff
@@ -139,313 +147,524 @@ export default {
-            />
-          </div>
-          <h1>
+      <div class="logo-container">
+        <div class="logo-box">
+          <LazyImage
+            :src="chart.icon"
+            class="logo"
+            :alt="t('catalog.charts.iconAlt', { app: chart.chartNameDisplay })"
+          />
+        </div>
+        <div
+          v-if="chart.featured"
+          v-clean-tooltip="t('generic.featured')"
+          class="featured-pill"
+        >
+          {{ t('generic.shortFeatured') }}
+        </div>
+      </div>
+      <div class="header-body">
+        <div class="header-top">
+          <h1
+            class="title"
+            data-testid="chart-header-title"
+          >
             <router-link :to="{ name: 'c-cluster-apps-charts' }">
               {{ t('catalog.chart.header.charts') }}:
             </router-link>
-            {{ chart.chartNameDisplay }} ({{ targetVersion }})
+            {{ chart.chartNameDisplay }}
           </h1>
+          <div
+            v-if="headerContent.statuses.length"
+            class="statuses"
+          >
+            <div
+              v-for="(status, i) in headerContent.statuses"
+              :key="i"
+              class="status"
+            >
+              <i
+                v-clean-tooltip="t(status.tooltip.key)"
+                :class="['icon', status.icon, status.color]"
+                :style="{color: status.customColor}"
+                role="img"
+                :aria-label="t(status.tooltip.key)"
+              />
+            </div>
+          </div>
+        </div>
+        <div class="header-sub-top">
+          <AppChartCardSubHeader :items="headerContent.subHeaderItems" />
+        </div>
+        <div class="header-middle">
+          <div
+            v-if="version && version.description"
+            class="description"
+          >
+            <p>{{ version.description }}</p>
+          </div>
+        </div>
+        <div class="header-bottom">
+          <AppChartCardFooter
+            :items="headerContent.footerItems"
+            @click:item="handleHeaderItemClick"
+          />
         </div>
-        <button
-          v-if="!requires.length"
-          type="button"
-          data-testid="btn-chart-install"
-          class="btn role-primary"
-          @click.prevent="install"
-        >
-          {{ t(`asyncButton.${action}.action` ) }}
-        </button>
       </div>
-      <div
-        v-if="chart.windowsIncompatible"
-        class="mt-5"
+      <button
+        v-if="!requires.length"
+        type="button"
+        data-testid="btn-chart-install"
+        class="btn role-primary"
+        @click.prevent="install"
       >
-        <label class="os-label">{{ t('catalog.charts.windowsIncompatible') }}</label>
-      </div>
-      <div
-        v-if="requires.length || warnings.length || targetedAppWarning || osWarning"
-        class="mt-20"
+        {{ t(`asyncButton.${action}.action` ) }}
```

Comment:
It looks like we're showing `upgrade` instead of the `update` in the figma.

<img width="126" height="54" alt="image" src="https://github.com/user-attachments/assets/4c0a6078-ff14-4d32-9cea-30e082d6e554" />